pkg/proto/nbcontract/v1/httpproxyconfig.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+package nbcontract.v1;
+
+message HTTPProxyConfig {
+  // HTTPProxy represents the proxy endpoint/URL using HTTP, that the traffic should go through for making HTTP connections outside the cluster
+  optional string HTTPProxy = 1;
+
+  // HTTPSProxy represents the proxy endpoint using HTTPS, that the traffic should go through for making HTTPS connections outside the cluster
+  optional string HTTPSProxy = 2;
+
+  // NoProxy represents a list of endpoints including IPs, domains, or other network CIDRs that should not be accessed through the proxy
+  repeated string NoProxy = 3;
```

Comment:
Part of me is wondering if naming it something like "NoProxyEntry" would make it clear it is not a full NO_PROXY string with separators ...

Reply:
inclined to keep as is, since there are comments highlighting the exact use, but can change if you have a strong preference for the name.